code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -115,6 +115,10 @@
 	e.start()
 	holder.clear_reagents()
 
+/datum/reagent/gunpowder/ghetto
+	name = "Improvised Gunpowder"
+	description = "Better than nothing i guess."
```

Comment:
capitalize your i's BiTCH

Reply:
alright fIkou

Reply:
unironically all of your prs have terrible grammar could you please review this stuff before you open it?

Reply:
unironically im not a native speaker, PR's are for a reason y'know?